llama_stack/providers/remote/memory/pgvector/pgvector.py

```diff
@@ -157,7 +157,7 @@ async def shutdown(self) -> None:
 
     async def register_memory_bank(
         self,
-        memory_bank: MemoryBankDef,
+        memory_bank: VectorMemoryBank,
```

Comment:
this signature (if you use my previous suggestion) should use that typed-tagged-union I suggested. each provider should get a "{ memory_bank_type: ..., <other_params> }" kind of an object and then it should decide it does not support that specific kind of memory bank

Reply:
fixed.